packages/react-client/src/__tests__/ReactFlight-test.js

```diff
@@ -29,8 +29,11 @@ describe('ReactFlight', () => {
     act = ReactNoop.act;
   });
 
-  function block(query, render) {
+  function block(render, query) {
     return function(...args) {
+      if (query === undefined) {
+        return [Symbol.for('react.server.block'), render];
+      }
       let curriedQuery = () => {
         return query(...args);
```

Comment:
Interestingly, if the query itself returns `undefined`, it will turn into `null` because of how JSON.stringify works on arrays. Fixing this would be kind of tricky but you're not really supposed to return `undefined` from a query anyway. We could warn for that same as we warn if you return undefined from render.

Reply:
> We could warn for that same as we warn if you return undefined from render.

yeah let's warn in that case